lib/ansible/modules/net_tools/basics/get_url.py

```diff
@@ -465,7 +465,7 @@ def main():
                 os.remove(checksum_tmpsrc)
                 lines = dict(s.split(None, 1) for s in lines)
                 filename = url_filename(url)
-                [checksum] = (k for (k, v) in lines.items() if v == filename)
+                [checksum] = (k for (k, v) in lines.items() if v.strip('./') == filename)
```

Comment:
Would it be better to use `lstrip()` just to be safe? I can't image a file name ending with `./` but I suppose that could be a valid file name.

Reply:
`/` is the standard file separator on UNIX, so a file can't possibly end with `/` because it would be interpreted as a directory... Maybe on Windows or other system although I would bet it's forbidden everywhere.